src/index.html

```diff
@@ -10,7 +10,21 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body data-qa="hover">
+    <!--First bar-->
+    <div class="search-bar search-bar__big" data-qa="big">
```

Comment:
```suggestion
    <form class="search-bar" data-qa="big">
```
use form tag instead div
search-bar__big element is redundant. 
set all styles for search-bar and just change necessary for small form in search-bar__small element.
same for other elements